### DISCUSSION TITLE
Question about accuracy of this tool

### DISCUSSION THREAD

**Discussion by XiangjunWu** [kyleskom/NBA-Machine-Learning-Sports-Betting#154] [General]
As I can read from the introduction, this tool uses the data of past games to predict the win rate of each teams in a game. For example all of a sudden all top players of ateam are leaving the team. Will this have an impact on the calculation of the winning rate?

Does this tool only considering the amount of games one team wins and loses or does it use other indicator as well for the predicting?

**Comment by kyleskom** [comment]
Please review the input data from the datasets folder. That is everything used to train. This takes into account team stats not player stats.

**Reply by XiangjunWu** [reply]
I see, thanks for the answer and also thanks for sharing this tool!
